src/software/world/field.cpp

```diff
@@ -20,7 +20,7 @@ Field Field::createSSLDivisionAField()
     return field;
 }
 
-Field Field::createField(TbotsProto::FieldType field_type)
+Field Field::createFieldProto(TbotsProto::FieldType field_type)
```

Comment:
this is returning a C++ Field object (not a proto that is generated by the protobuf compiler).